readme.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+- [DEMO LINK](https://<nikolaygit1>.github.io/layout_hello-world/)
+- [TEST REPORT LINK](https://<nikolaygit1>.github.io/layout_hello-world/report/html_report/)
```

Comment:
in future paste this text into PR's description 

```suggestion
- [DEMO LINK](https://nikolaygit1.github.io/layout_hello-world/)
- [TEST REPORT LINK](https://nikolaygit1.github.io/layout_hello-world/report/html_report/)
```

Reply:
also please deploy (step 18)

https://mate-academy.github.io/layout_task-guideline/